apps/src/code-studio/components/ImagePicker.jsx

```diff
@@ -11,7 +11,9 @@ const extensionFilter = {
   // Note: .jfif files will be converted to .jpg by the server.
   image: '.jpg, .jpeg, .jfif, .gif, .png',
   audio: '.mp3',
-  document: '.jpg, .jpeg, .gif, .png, .pdf, .doc, .docx'
+  document: '.jpg, .jpeg, .gif, .png, .pdf, .doc, .docx',
+  // All valid extensions (used if type filter is not specified)
+  all: '.jpg, .jpeg, .jfif, .gif, .png, .mp3, .pdf, .doc, .docx'
```

Comment:
is there a case where `all` is used?

Reply:
Yeah, so `all` is used when `typeFilter` is not provided. So far the only instance of this I could find was in `SettingsCog.tsx` [here](https://github.com/code-dot-org/code-dot-org/blob/3ffb1da0776c4c08c00192da35b9fd520e277caf/apps/src/lib/ui/SettingsCog.jsx#L46). This is needed because if `allowedExtensions` is not specified at all, then any type of file can be uploaded - that being said only files that are whitelisted by `asset_bucket.rb` will actually be uploaded without error, but because I added .wav to that whitelist, I added this to prevent .wav files from being uploaded anywhere but Javalab.

Reply:
should we instead use `default` here? It may be confusing in the future that `all` does not include all extensions (such as .wav)

Reply:
Yeah, that makes sense, will update.